### PR TITLE
[IMP] Add corporate cla opa-consulting

### PR DESCRIPTION
**Pull request by cormaza**
Add corporate cla opa-consulting

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jco-odoo** [comment]
Should not we delete cormaza.md then ?  @mart-e  

Maybe also do the commit message with [CLA] but that is more of a detail.

**Comment by jco-odoo** [comment]
Should not we delete cormaza.md then ?  @mart-e  

Maybe also do the commit message with [CLA] but that is more of a detail.

File: doc/cla/corporate/opa-consulting.md [hunk 1]
```diff
@@ -9,8 +9,11 @@ declaration.
 Signed,
 
 Guenael Labois  glabois@opa-consulting.com https://github.com/Stentor
+Christopher Ormaza cormaza@opa-consulting.com https://github.com/opa-cormaza
```
Comment (jco-odoo) [inline doc/cla/corporate/opa-consulting.md:12]: Not necessary to be in signed and in the contributors also.  Could be :

```suggestion
Signed
Guenael Labois  glabois@opa-consulting.com https://github.com/Stentor - until 2021-07-31

List of contributors:

Guenael Labois  glabois@h4o-studio.com https://github.com/Stentor - until 2021-07-31
Christopher Ormaza cormaza@opa-consulting.com https://github.com/opa-cormaza
```

**Comment by jco-odoo** [comment]
needs to target saas-14.5

**Comment by mart-e** [comment]
> needs to target saas-14.5

If you wish, the easiest is to target the CLA with the lowest supported branch (i.e. 12.0) and we will forward port the CLA up to master.

**Pull request closed without merging**